test.js

```diff
@@ -6,5 +6,7 @@ const { execSync } = require('child_process')
 fs.readdirSync('test')
   .filter(file => file.endsWith('.js') || !file.includes('.'))
   .forEach(file => {
-    require('./test/' + file)
+    if (!file.includes('wasm')) {
```

Comment:
Why exclude it?

Reply:
It was a leftover from when I was building wasm after the other crates; fixed